evalml/objectives/standard_metrics.py

```diff
@@ -345,6 +345,17 @@ def objective_function(self, y_true, y_predicted, X=None):
         return metrics.mean_absolute_error(y_true, y_predicted)
 
 
+class MAPE(RegressionObjective):
+    """Mean absolute percentage error for time series regression."""
+    name = "Mean Absolute Percentage Error"
+    greater_is_better = False
+    score_needs_proba = False
```

Comment:
nit-pick: You don't need to specify `score_needs_proba` since that is specified in `RegressionObjective`